pkg/build/api/types.go

```diff
@@ -55,6 +55,9 @@ type BuildParameters struct {
 
 	// Output describes the Docker image the Strategy should produce.
 	Output BuildOutput `json:"output,omitempty"`
+
+	// Compute resource requirements to execute the build
+	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
> On Apr 21, 2015, at 11:10 PM, Ben Parees <notifications@github.com> wrote:
> 
> In pkg/build/api/types.go:
> 
> > @@ -55,6 +55,9 @@ type BuildParameters struct {
> >  
> >  	// Output describes the Docker image the Strategy should produce.
> >  	Output BuildOutput `json:"output,omitempty"`
> > +
> > +	// Compute resource requirements to execute the build
> > +	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
> Both have the exact same meaning. No limits, no requests.
> 
> both will be enforced that way, but one is a hanging chad with guessed intent and which should elicit a warning "hey it looks like you tried to define a quota but never finished, you might want to take a look at that"
> 
We've said elsewhere we don't insert pointer unless the zero value can't be discriminated from empty for a user.  The serializer in go is the problem - it only handles omitempty correctly for a set of known types.  I expect we'll fix that over time, in which case we could make the deserializer smarter.
> —
> Reply to this email directly or view it on GitHub.
>